config/bidderinfo.go

```diff
@@ -356,6 +358,50 @@ func validateMaintainer(info *MaintainerInfo, bidderName string) error {
 	return nil
 }
 
+func validateAliasCapabilities(bidder BidderInfo, infos BidderInfos, bidderName string) error {
+	if len(bidder.AliasOf) == 0 {
+		return nil
+	}
+
+	parentBidder, parentFound := infos[bidder.AliasOf]
+	if !parentFound {
+		return fmt.Errorf("bidder: %s not found for an alias: %s", bidder.AliasOf, bidderName)
+	}
```

Comment:
I think we should just validate the capabilities part here. This check is already covered in https://github.com/prebid/prebid-server/blob/92211a53da0a3a1dcf80c64c792f36265d88ffcb/config/bidderinfo.go#L331-L342 
At this point, that check should have already been called. So, no need for re-check and safely use `parentBidder := infos[bidder.AliasOf]`